cmd/dependabot/internal/cmd/test.go

```diff
@@ -37,22 +37,25 @@ var testCmd = &cobra.Command{
 		processInput(&scenario.Input)
 
 		if err := infra.Run(infra.RunParams{
-			CacheDir:      cache,
-			Creds:         scenario.Input.Credentials,
-			Debug:         debugging,
-			Expected:      scenario.Output,
-			ExtraHosts:    extraHosts,
-			InputName:     file,
-			InputRaw:      inputRaw,
-			Job:           &scenario.Input.Job,
-			LocalDir:      local,
-			Output:        output,
-			ProxyCertPath: proxyCertPath,
-			ProxyImage:    proxyImage,
-			PullImages:    pullImages,
-			Timeout:       timeout,
-			UpdaterImage:  updaterImage,
-			Volumes:       volumes,
+			CacheDir:            cache,
+			CollectorConfigPath: collectorConfigPath,
+			CollectorImage:      collectorImage,
+			Creds:               scenario.Input.Credentials,
+			Debug:               debugging,
+			EnableOpenTelemetry: enableOpenTelemetry,
```

Comment:
Is there ever a time we'd need to `enable-opentelemetry` but not have a `collector-config`? 

If no, then can we remove `enable-opentelemetry` and assume if a `collector-config` was passed then it is enabled? Does that make sense?

Reply:
I can't see a reason why. Are you okay with changing the `if params.EnableOpenTelemetry == true` to `if params.CollectorConfig != ""`?

Reply:
Yes!